docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -44,21 +48,34 @@ This includes:
 ![Diagram of a high-availability Teleport
 architecture](../../img/deploy-a-cluster/teleport-ha-architecture.png)
 
-## Layer 4 load balancer
+## Layer 4 load balancers
 
-The load balancer forwards traffic from users and services to an available
-Teleport instance. This must not terminate TLS, and must transparently forward
-the TCP traffic it receives. In other words, this must be a Layer 4 load
-balancer, not a Layer 7 (e.g., HTTP) load balancer. 
+High-availability Teleport clusters require two load balancers:
+- **Proxy Service load balancer:** A load balancer to receive traffic from
+  outside the network where your Teleport cluster is running and forward it to
+  an available Proxy Service instance. This load balancer handles TCP traffic
+  from users and services in a variety of application-layer protocols.
+- **Auth Service load balancer:** A load balancer to forward traffic from a
+  Proxy Service instance to an available Auth Service instance. This handles TLS
+  traffic to the Auth Service's gRPC endpoint. It can also handle traffic to
+  Teleport Kubernetes Service instances running in the Auth Service pool in
+  order to enable secure access to your Teleport cluster's infrastructure.
 
-We recommend configuring your load balancer to route traffic across multiple
+Neither load balancer must terminate TLS. Both load balancers must transparently
```

Comment:
"Neither load balancer must" makes it sound like terminating TLS is optional and the LB doesn't have to, but can, terminate TLS.

Should we be stricter here and say "You must not terminate TLS on either of these load balancers"??

(That said, we technically do support ALBs for most protocols now..)

Reply:
Paging @greedy52 who can answer this better than I can

Reply:
Afaik:
- Proxy -> Auth LB must not terminate TLS
- Client -> Proxy LB can terminate TLS since 13.0 (but this might come with additional restrictions for kube access)

Reply:
Thanks! What additional restrictions are these? Is there anything else I should know,  @greedy52?

Reply:
As mentioned, it doesn't support all protocols yet and it also has some performance penalties which we may do some performance improvement in v14. So I think layer 4 LBs should still be recommended way to set up HA.

I was planning to write up a separate page for TLS Routing behind layer 7 LB preview. Maybe I can add a note here to link to that page when it's available?

Reply:
Sounds good! I'll keep the recommendation the way it is for now and we can add the note when the L7 guide is ready.

Reply:
I think "Both..must not" is ambiguous, since it can also mean "one can". I've suggested: 

```
Both load balancers must transparently forward the TCP traffic they receive,
without terminating TLS. In other words, these must be Layer 4 load balances,
not Layer 7 (e.g., HTTP) load balancers.
```